docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,76 @@
+
+### PostgreSQL monitoring performance improvement
+
+In our continuing efforts to improve PMM, we recognize the significance of performance and its impact on our users. Starting with PMM 2.41.0, we have optimized resource usage for PostgreSQL users. This has been accomplished by reducing the number of connections used in monitoring.
+
+### PMM DBaaS functionality evolution into Percona Everest
+
+We have achieved positive outcomes in developing PMM's Database-as-a-Service feature. As we move to the next level, we have decided to separate the DBaaS functionality into an independent product. Consequently, we are discontinuing the DBaaS functionality in PMM and offering a [migration path to Everest](http://per.co.na/pmm-to-everest-guide).
+
+While the DBaaS functionality will remain available in PMM v2, all future updates and enhancements will be exclusively accessible through the Percona Everest interface.
+We strongly recommend you try [Percona Everest](http://per.co.na/pmm-to-everest) for a more streamlined and robust experience with database deployments.
+
+## New Features
+
+[PMM-8655](https://jira.percona.com/browse/PMM-8655) - Alongside Victoria metrics, PMM now also captures QAN data during network outages. Once the connection is restored, all accumulated data is promptly delivered, ensuring you have comprehensive information to work with across your instances.
+[PMM-12459](https://jira.percona.com/browse/PMM-12459) - The [pmm_dump client utility](https://docs.percona.com/pmm-dump-documentation/index.html) previously available as a standalone client for PMM Server is now readily accessible within the PMM user interface. 
+[PMM-12463](https://jira.percona.com/browse/PMM-12463) - [Telemetry] Collecting information about env variables usage in PMM
+
+## Improvements
+
+[PMM-11341](https://jira.percona.com/browse/PMM-11341) - PMM 2.41.0 introduces limit for Auto-discovery in PostgreSQL, a feature that dynamically discovers all databases in your PostgreSQL instance. Limiting Auto-discovery reduces connections and prevents high CPU and RAM usage caused by multiple databases.
+[PMM-12375](https://jira.percona.com/browse/PMM-12375) - Starting with PMM 2.41.0, each instance of a service gets a `version` attribute to the response of the endpoint that provides a list of services being monitored by PMM.
+[PMM-12422](https://jira.percona.com/browse/PMM-12422) - Starting with PMM 2.41.0, a new flag called `--expose-exporter-address` has been introduced. When you enable this flag and set the metrics mode to **push**, the node exporter for that agent will use the IP address `0.0.0.0`. This means that any IP address, either from a local system or from anywhere on the internet, can access it. If the flag is not enabled, the node exporter will use the address `127.0.0.1`, the default name for the computer you are working on, or the localhost.
```

Comment:
```suggestion
[PMM-12422](https://jira.percona.com/browse/PMM-12422) - Starting with PMM 2.41.0, a new flag called `--expose-exporter-address` has been introduced. When you enable this flag any IP address, either from a local system or from anywhere on the internet, can access exporter endpoints. If the flag is not enabled, the exporter will be available only locally.
```
I think we can remove technical details about 0.0.0.0 here and just leave the part about being available from anywhere and localhost.